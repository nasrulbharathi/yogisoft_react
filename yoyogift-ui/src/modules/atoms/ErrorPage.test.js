import React from "react";
import { shallow } from "enzyme";
import ErrorPage from "./ErrorPage";

describe("Error Page", () => {
  it("should be defined", () => {
    expect(ErrorPage).toBeDefined();
  });
  it("should render ErrorPage", () => {
    const component = shallow(<ErrorPage />);
    expect(component).toMatchSnapshot();
  });
});
