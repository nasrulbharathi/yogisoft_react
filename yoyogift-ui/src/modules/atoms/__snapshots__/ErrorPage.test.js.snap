// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render ErrorPage 1`] = `ShallowWrapper {}`;
